Gemfile

```diff
@@ -42,15 +42,14 @@ gem 'slack-notifier', '~> 2.3', '>= 2.3.2'
 # Local previewing
 gem 'sassc'
 gem 'sinatra', '~> 2', '>= 2.0.8.1'
+gem 'thin'
```

Comment:
Does `sinatra` automatically use `thin` when it's available?

Reply:
Yeah—I think ruby3 no longer includes webrick? Sinatra will use a web server it can find—including webrick, puma, thin... I found thin mentioned in the documentation, so chose that, completely arbitrarily.